src/components/styles/_light-dark-theme.scss

```diff
@@ -0,0 +1,34 @@
+@import "./colors";
+
+$themes: (
+  bgThemeColor1: (
+    darkTheme: $blue700,
+    lightTheme: $grey100,
+  ),
+  bgThemeColor2: (
+    darkTheme: $blue600,
+    lightTheme: $grey200,
+  ),
+  textThemeColor1: (
+    darkTheme: $grey100,
+    lightTheme: $black,
+  ),
+);
```

Comment:
duplicate lines of code (same like into the `src/components/styles/_colors.scss`)